front-end/src/Header.js

```diff
@@ -0,0 +1,25 @@
+import 'react-sticky-header/styles.css';
+import StickyHeader from 'react-sticky-header';
+import './Header.css' 
+import Hamburger from './Hamburger'
+
+const Header = () => (
+  [
+  <Hamburger pageWrapId={'page-wrap'} outerContainerId={'outer-container'} />,
+  <StickyHeader    
+    // This section is sticky 
+    header={
+      <div className="Header_root">
+        <h1 className="Header_title">TV Tracker</h1>
+      </div>
```

Comment:
I think `Hamburger` should be placed within this `header` attribute? I've played around with this a bit on my own, and putting it in there and altering the `top` style of `.bm-burger-button` aligns the burger menu nicely with the text (somewhere between 1em and 2em looks nice, imo).

Also a bit minor, but I'd change the class names from `Header_root` and `Header_title` to `header-root` and `header-title` to be consistent with our class names we've made.

Reply:
When I try this, the hamburger menu pops out beneath the level of the header text. If you know a way to fix this issue lmk